boards/microchip/mpfs_icicle/mpfs_icicle_polarfire_e51_defconfig

```diff
@@ -13,3 +13,4 @@ CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
 # GPIO driver options
 CONFIG_GPIO=y
 CONFIG_I2C=y
+CONFIG_CAN=y
```

Comment:
The boards should not enable CAN per default, only applications needing it should enable it.

Reply:
@con-pax Please have a look at this comment.